index.html

```diff
@@ -0,0 +1,218 @@
+             both academy and industry by covering not only basic
+             and applied research,
+             but also philosophical and ethical issues
+             regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/exhibition-icon.png"
+                alt="exhibition"
+                class="icon"
+              />
+              <p class="program-name">CC Exhibition</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry 
+             by covering not only basic and applied research,
+             but also philosophical and ethical issues
+              regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/forum-icon.png" alt="forum" class="icon" />
+              <p class="program-name">Forum</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical 
+             issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/workshop-icon.png"
+                alt="workshop"
+                class="icon"
+              />
+              <p class="program-name">Workshop</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical
+              issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/ignite-icon.png" alt="ignite" class="icon" />
+              <p class="program-name">CC Ignite</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
```

Comment:
Please follow the design for `Featured speakers` and `partner` sections on the Desktop view.
Your:
![partner-y](https://user-images.githubusercontent.com/35504149/137373498-572850b8-f7d4-4a14-9f05-ad7043fb84f9.png)
Design:

![partners](https://user-images.githubusercontent.com/35504149/137373512-05c28ee0-40aa-4d0c-a752-5c34974cfece.png)
Your:
![feature-y](https://user-images.githubusercontent.com/35504149/137373350-2ccbdeb0-5ea4-4079-8a1c-43046ff58142.png)
Design:
![feature-o](https://user-images.githubusercontent.com/35504149/137373385-def42e16-e96e-4447-9b36-9cf03d43ee8a.png)